src/shared/utils/provider.ts

```diff
@@ -4,25 +4,28 @@ import config from 'config'
 import bitbucketLogo from 'assets/providers/bitbucket-icon.svg'
 import githubLogo from 'assets/providers/github-icon.svg'
 import gitlabLogo from 'assets/providers/gitlab-icon.svg'
+import { Provider } from 'shared/api/helpers'
 
-export function providerToName(provider) {
-  return {
-    gh: 'Github',
-    bb: 'BitBucket',
-    gl: 'Gitlab',
-    ghe: 'Github Enterprise',
-    gle: 'Gitlab Enterprise',
-    bbs: 'BitBucket Server',
-    github: 'Github',
-    bitbucket: 'BitBucket',
-    gitlab: 'Gitlab',
-    github_enterprise: 'Github Enterprise',
-    gitlab_enterprise: 'Gitlab Enterprise',
-    bitbucket_server: 'BitBucket Server',
-  }[provider.toLowerCase()]
+export function providerToName(provider: Provider) {
+  return (
+    {
+      gh: 'Github',
+      bb: 'BitBucket',
+      gl: 'Gitlab',
+      ghe: 'Github Enterprise',
+      gle: 'Gitlab Enterprise',
+      bbs: 'BitBucket Server',
+      github: 'Github',
+      bitbucket: 'BitBucket',
+      gitlab: 'Gitlab',
+      github_enterprise: 'Github Enterprise',
+      gitlab_enterprise: 'Gitlab Enterprise',
+      bitbucket_server: 'BitBucket Server',
+    }[provider.toLowerCase()] || 'Github'
```

Comment:
Added a default || 'Github' here to quell some TS errors below about this function potentially returning undefined

Reply:
Actually removing this on my local and doing some other stuff